src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -133,7 +134,7 @@ public AbstractInstances(final Storage storage, final HttpClient client) {
     CollectionResourceRepository precedingSucceedingTitlesRepository =
       new CollectionResourceRepository(precedingSucceedingTitlesClient);
 
-    List<String> instanceId = Arrays.asList(instance.getId());
+    List<String> instanceId = Collections.singletonList( instance.getId() );
```

Comment:
```suggestion
    List<String> instanceId = List.of(instance.getId());
```

Reply:
Not that it matters to me, but Collections.singletonList was chosen to be consistent with the method used for this elsewhere in mod-inventory.